node/_readline.d.ts

```diff
@@ -1,43 +1,54 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 // deno-lint-ignore-file no-explicit-any
 
-// Forked from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e4ebac60727c5a6fd0b99f615b48270f55c40284/types/node/readline.d.ts
+// Forked from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cd61f5b4d3d143108569ec3f88adc0eb34b961c4/types/node/readline.d.ts
 
 import { Abortable, EventEmitter } from "./_events.d.ts";
+import * as promises from "./readline/promises.ts";
 import { ReadableStream, WritableStream } from "./_global.d.ts";
 import { Buffer } from "./buffer.ts";
+import type {
+  AsyncCompleter,
+  Completer,
+  CompleterResult,
+  ReadLineOptions,
+} from "./_readline_2.d.ts";
```

Comment:
Can you give this new file a more meaningful name. Something like `_readline_shared_types.d.ts` for example.

Reply:
Sorry, I couldn't name it appropriately. The file name has been revised. (eeb6ce0)